examples/GetTrainingModelProgress.java

```diff
@@ -0,0 +1,31 @@
+import com.indico.IndicoClient;
+import com.indico.IndicoConfig;
+import com.indico.entity.Model;
+import com.indico.entity.ModelGroup;
+import com.indico.query.ModelGroupQuery;
+import com.indico.query.TrainingModelWithProgressQuery;
+import java.io.IOException;
+
+public class GetTrainingModelProgress {
+
+    public static void main(String args[]) throws IOException {
+        IndicoConfig config = new IndicoConfig.Builder()
+                .host("dev.indico.io")
+                .tokenPath("__TOKEN_PATH__")
+                .build();
+
+        try (IndicoClient client = new IndicoClient(config)) {
+            ModelGroupQuery modelGroupQuery = client.modelGroupQuery();
+            TrainingModelWithProgressQuery trainingModelWithProgress = client.trainingModelWithProgressQuery();
+            ModelGroup modelGroup = modelGroupQuery.id(__MODEL_ID__).query();
```

Comment:
Call these __MODEL_GROUP_ID__ as model id will be confuse with selected model inside a model group.

Reply:
Fixed.